api/src/test/java/marquez/DatasetIntegrationTest.java

```diff
@@ -574,4 +576,58 @@ public void testApp_doesNotShowDeletedDatasetAfterUndeleteNamespace() throws IOE
     jobs = client.listJobs(namespaceName);
     assertThat(jobs).hasSize(1);
   }
+
+  @Test
+  public void testApp_getTableVersionsWithSymlinks() {
+    client.createDataset(NAMESPACE_NAME, DB_TABLE_NAME, DB_TABLE_META);
+
+    ImmutableMap<String, Object> outputFacets =
+        ImmutableMap.of("outputFacetKey", "outputFacetValue");
+    ImmutableMap<String, Object> inputFacets = ImmutableMap.of("inputFacetKey", "inputFacetValue");
+
+    final LineageEvent.DatasetFacets datasetFacets =
+        LineageTestUtils.newDatasetFacet(
+            outputFacets,
+            LineageEvent.SchemaField.builder()
+                .name("firstname")
+                .type("string")
+                .description("the first name")
+                .build());
+    datasetFacets
+        .getDocumentation()
+        .setDescription(DB_TABLE_META.getDescription().orElse("the dataset documentation"));
+    datasetFacets.setSymlinks(
+        new LineageEvent.DatasetSymlinkFacet(
+            PRODUCER_URL,
+            SCHEMA_URL,
+            Collections.singletonList(
+                new LineageEvent.SymlinkIdentifier("symlinkNamespace", "symlinkName", "type"))));
+    final LineageEvent lineageEvent =
+        LineageEvent.builder()
+            .producer("testApp_getTableVersionsWithSymlinks")
+            .eventType("COMPLETE")
+            .run(
+                new LineageEvent.Run(
+                    UUID.randomUUID().toString(), LineageEvent.RunFacet.builder().build()))
+            .job(LineageEvent.Job.builder().namespace(NAMESPACE_NAME).name(JOB_NAME).build())
+            .eventTime(ZonedDateTime.now())
+            .inputs(
+                Collections.singletonList(
+                    LineageEvent.Dataset.builder()
+                        .namespace(NAMESPACE_NAME)
+                        .name(DB_TABLE_NAME)
+                        .facets(datasetFacets)
+                        .build()))
+            .outputs(Collections.emptyList())
+            .build();
+    System.out.println("event is " + lineageEvent);
```

Comment:
Remove log statement here.

Reply:
i pushed a new version removing the log statement